tempodb/tempodb.go

```diff
@@ -544,3 +546,28 @@ func (rw *readerWriter) compactedBlocklist(tenantID string) []*encoding.Compacte
 
 	return copiedBlocklist
 }
+
```

Comment:
I'm not sure if my approach is idiomatic!

I created a tenantSet from the backendTenants slide. Later I loop over all the tenants on the in-memory blocklists, and If they don't exist on the tenantSet, I remove them.